tsconfig.json

```diff
@@ -1,16 +1,13 @@
 {
   "include": ["*.js", "script/**/*.js", "test/**/*.js"],
   "compilerOptions": {
-    "target": "ES2020",
-    "lib": ["ES2020"],
-    "module": "ES2020",
-    "moduleResolution": "node",
-    "resolveJsonModule": true,
+    "target": "es2021",
+    "lib": ["es2021"],
+    "module": "node16",
     "allowJs": true,
```

Comment:
Technically unrelated, but I wouldn't mind if you delete `allowJs` while you're at it. :)

Reply:
Good idea!
Removed in [c898b77](https://github.com/rehypejs/rehype/pull/123/commits/c898b7799fee0730b576d5a8e838c5c49357ab13)